src/pages/CubePlaytestPage.js

```diff
@@ -345,7 +345,7 @@ const CubePlaytestPage = ({ cube, cubeID, canEdit, decks, draftFormats }) => {
 
 CubePlaytestPage.propTypes = {
   cube: PropTypes.shape({
-    cards: PropTypes.arrayOf(PropTypes.object).isRequired,
+    cards: PropTypes.arrayOf(PropTypes.object),
```

Comment:
Why this change?

Reply:
I was getting warnings in the browser that we were passing a cube without cards and I checked and we aren't using cards in this file so it didn't seem to need to be required.